### DISCUSSION TITLE
[Bug] LobeChat 默认助理的默认配置中model改为model=gemini-1.5-flash-latest，首次访问仍会访问openai接口

### DISCUSSION THREAD

**Discussion by baiyi-os** [lobehub/lobehub#2882] [General Question] [answered]
### 🐛 问题描述

[Bug] LobeChat 默认助理的默认配置中model改为model=gemini-1.5-flash-latest，首次访问仍会访问openai接口

### 📷 复现步骤

![11](https://github.com/lobehub/lobe-chat/assets/58473561/c3d17e02-eb55-43ff-8b07-a8e4ee4438fe)

[a comment by an automated account is not shown]

**Comment by RubuJam** [comment, marked as answer]
完整设置为：`model=gemini-1.5-flash-latest;provider=google;`

The complete setup is: `model=gemini-1.5-flash-latest;provider=google;`

[a comment by an automated account is not shown]

**Comment by baiyi-os** [comment]
可以了，十分感谢

[a comment by an automated account is not shown]
